http/src/main/resources/swagger/api-docs.yaml

```diff
@@ -2170,86 +2554,86 @@ components:
     GceConfig:
       description: Configuration for Google Compute Engine instances.
       allOf:
-      - $ref: '#/components/schemas/RuntimeConfig'
```

Comment:
Hm.. was spacing wrong here? I touched this recently, would be good to make sure it still displays correctly